core/primitives/src/stateless_validation.rs

```diff
@@ -94,6 +97,10 @@ impl PartialEncodedStateWitness {
         self.inner.part_ord
     }
 
+    pub fn part_len(&self) -> usize {
```

Comment:
part_size? or data_size?

Reply:
+1, had the same comment, let's use the term "size" instead of "len" everywhere